### PR TITLE
choco/*: add Polish translation

### PR DESCRIPTION
**Pull request by mpiasecki1**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pl/windows/choco-apikey.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# choco-apikey
```
Comment (navarroaxel) [inline pages.pl/windows/choco-apikey.md:1]:
```suggestion
# choco apikey
```

**Review by navarroaxel — APPROVED** [review]

File: pages.pl/windows/choco-feature.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# choco feature
+
+> Zarządzanie dostępnymi funkcjani Chocolatey:
```
Comment (kbdharun) [inline pages.pl/windows/choco-feature.md:3]:
```suggestion
> Zarządzanie dostępnymi funkcjani Chocolatey.
```

File: pages.pl/windows/choco-list.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# choco list
+
+> Wyświetlanie lusty pakietów Chocolatey.
```
Comment (kbdharun) [inline pages.pl/windows/choco-list.md:3]: Shouldn't it be `lista`?

File: pages.pl/windows/choco-search.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# choco search
+
+> Wyszukiwanie pakietów Chocolatey dostępnych lokalnie lub w zdalnych źródłach/repozytoriach.
+> Więcej informacji: <https://chocolatey.org/docs/commands-search>.
+
+- Wyszkukiwanie pakietów:
+
+`choco search {{kwerenda(szukana_fraza)}}`
```
Comment (kbdharun) [inline pages.pl/windows/choco-search.md:8]:
```suggestion
`choco search {{kwerenda}}`
```

File: pages.pl/windows/choco-search.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Wyszkukiwanie lokalnych pakietów:
+
+`choco search {{kwerenda(szukana_fraza)}} --local-only`
```
Comment (kbdharun) [inline pages.pl/windows/choco-search.md:12]:
```suggestion
`choco search {{kwerenda}} --local-only`
```

File: pages.pl/windows/choco-search.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- Wyświetlanie wyłącznie dokładnych dopasowań do podanej kwerendy/szukanej frazy:
+
+`choco search {{kwerenda(szukana_fraza)}} --exact`
```
Comment (kbdharun) [inline pages.pl/windows/choco-search.md:16]:
```suggestion
`choco search {{kwerenda}} --exact`
```

File: pages.pl/windows/choco-search.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+- Automatyczna akceptacja wszystkich monitów (--yes):
+
+`choco search {{kwerenda(szukana_fraza)}} --yes`
```
Comment (kbdharun) [inline pages.pl/windows/choco-search.md:20]:
```suggestion
`choco search {{kwerenda}} --yes`
```

File: pages.pl/windows/choco-search.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+- Ustawienie określonego źródła do wyszukiwania pakietów:
+
+`choco search {{kwerenda(szukana_fraza)}} --source {{adres_url|alias}}`
```
Comment (kbdharun) [inline pages.pl/windows/choco-search.md:24]:
```suggestion
`choco search {{kwerenda}} --source {{adres_url|alias}}`
```

File: pages.pl/windows/choco-search.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+- Podanie nazwy użytkownika i hasła do uwierzytelnienia:
+
+`choco search {{kwerenda(szukana_fraza)}} --user {{nazwa_użytkownika}} --password {{hasło}}`
```
Comment (kbdharun) [inline pages.pl/windows/choco-search.md:28]:
```suggestion
`choco search {{kwerenda}} --user {{nazwa_użytkownika}} --password {{hasło}}`
```

File: pages.pl/windows/choco-source.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+- Wylistowanie aktualnie dostępmnych źródeł:
+
+`choco source list`
+
+- Dodanie nowego źródła:
+
+`choco source add --name {{nazwa}} --source {{adres_url}}`
+
+- Dodanie nowego źródła z użyciem poświadczeń (nazwa użytkownika i hasło):
```
Comment (kbdharun) [inline pages.pl/windows/choco-source.md:14]:
```suggestion
- Dodanie nowego źródła z użyciem poświadczeń:
```
Credentials is self explanatory.

File: pages.pl/windows/choco-upgrade.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+
+- Aktualizacja pakietu do konkretnej wersji:
+
+`choco upgrade {{nazwa_pakietu(pakietów)}}`
+
+- Aktualizacja wszystkich pakietów zarządzanych przez Chocolatey:
+
+`choco upgrade all`
+
+- Aktualizacja wszystkich pakietów zarządzanych przez Chocolatey z wyjątkiem podanych w parametrze --except:
```
Comment (kbdharun) [inline pages.pl/windows/choco-upgrade.md:18]: Rephrase this line to better align with the main page. (Upgrade all except specified comma-separated packages:)

File: pages.pl/windows/choco-upgrade.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+> Aktualizacja jednej lub więcej paczek zarządzanych przez Chocolatey.
+> Więcej informacji: <https://chocolatey.org/docs/commands-upgrade>.
+
+- Aktualizacja jednego lub więcej pakietów (oddzielonych spacją):
+
+`choco upgrade {{nazwa_pakietu(pakietów)}}`
+
+- Aktualizacja pakietu do konkretnej wersji:
+
+`choco upgrade {{nazwa_pakietu(pakietów)}}`
```
Comment (kbdharun) [inline pages.pl/windows/choco-upgrade.md:12]:
```suggestion
`choco upgrade {{pakiet(pakietów)}}`
```

File: pages.pl/windows/choco-upgrade.md [hunk 13]
```diff
@@ -0,0 +1,32 @@
+# choco upgrade
+
+> Aktualizacja jednej lub więcej paczek zarządzanych przez Chocolatey.
+> Więcej informacji: <https://chocolatey.org/docs/commands-upgrade>.
+
+- Aktualizacja jednego lub więcej pakietów (oddzielonych spacją):
+
+`choco upgrade {{nazwa_pakietu(pakietów)}}`
```
Comment (kbdharun) [inline pages.pl/windows/choco-upgrade.md:8]:
```suggestion
`choco upgrade {{pakiet(pakietów)}}`
```

File: pages.pl/windows/choco-upgrade.md [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+
+`choco upgrade all`
+
+- Aktualizacja wszystkich pakietów zarządzanych przez Chocolatey z wyjątkiem podanych w parametrze --except:
+
+`choco upgrade all --except "{{nazwa_pomijanego_pakietu(pakietów)}}"`
+
+- Automatyczna akceptacja wszystkich monitów podczas aktualizacji pakietu (--yes):
+
+`choco upgrade {{nazwa_pakietu(pakietów)}} --yes`
```
Comment (kbdharun) [inline pages.pl/windows/choco-upgrade.md:24]:
```suggestion
`choco upgrade {{pakiet(pakietów)}} --yes`
```

File: pages.pl/windows/choco-upgrade.md [hunk 15]
```diff
@@ -0,0 +1,32 @@
...
+
+- Ustawienie określonego źródła/repozytorium pakietów:
+
+`choco upgrade {{nazwa_pakietu(pakietów)}} --source {{adres_url|alias}}`
```
Comment (kbdharun) [inline pages.pl/windows/choco-upgrade.md:28]:
```suggestion
`choco upgrade {{pakiet(pakietów)}} --source {{adres_url|alias}}`
```

File: pages.pl/windows/choco-upgrade.md [hunk 16]
```diff
@@ -0,0 +1,32 @@
...
+
+- Podanie nazwy użytkownika i hasła do uwierzytelnienia:
+
+`choco upgrade {{nazwa_pakietu(pakietów)}} --user {{nazwa_użytkownika}} --password {{hasło}}`
```
Comment (kbdharun) [inline pages.pl/windows/choco-upgrade.md:32]:
```suggestion
`choco upgrade {{pakiet(pakietów)}} --user {{nazwa_użytkownika}} --password {{hasło}}`
```

File: pages.pl/windows/choco-uninstall.md [hunk 17]
```diff
@@ -0,0 +1,24 @@
...
+
+`choco uninstall {{nazwa_pakietu(pakietów)}}`
+
+- Odinstalowanie konkretnej wersji pakietu:
+
+`choco uninstall {{nazwa_pakietu}} --version {{wersja}}`
+
+- Automatyczna akceptacja wszystkich monitów podczas deinstalacji pakietu (--yes):
+
+`choco uninstall {{nazwa_pakietu(pakietów)}} --yes`
```
Comment (kbdharun) [inline pages.pl/windows/choco-uninstall.md:16]:
```suggestion
`choco uninstall {{pakiet}} --yes`
```

File: pages.pl/windows/choco-uninstall.md [hunk 18]
```diff
@@ -0,0 +1,24 @@
...
+
+- Odinstalowanie wszystkich zależności podczas procesu deinstalacji danego pakietu/pakietów:
+
+`choco uninstall {{nazwa_pakietu(pakietów)}} --remove-dependencies`
```
Comment (kbdharun) [inline pages.pl/windows/choco-uninstall.md:20]:
```suggestion
`choco uninstall {{pakiet}} --remove-dependencies`
```

File: pages.pl/windows/choco-uninstall.md [hunk 19]
```diff
@@ -0,0 +1,24 @@
+# choco uninstall
+
+> Odinstalowanie jednego lub więcej pakietów zarządzanych przez Chocolatey.
+> Więcej informacji: <https://chocolatey.org/docs/commands-uninstall>.
+
+- Odinstalowanie jednego lub więcej pakietów (oddzielonych spacją):
+
+`choco uninstall {{nazwa_pakietu(pakietów)}}`
```
Comment (kbdharun) [inline pages.pl/windows/choco-uninstall.md:8]:
```suggestion
`choco uninstall {{{pakiet(pakietów)}}`
```

File: pages.pl/windows/choco-uninstall.md [hunk 20]
```diff
@@ -0,0 +1,24 @@
...
+
+- Odinstalowanie konkretnej wersji pakietu:
+
+`choco uninstall {{nazwa_pakietu}} --version {{wersja}}`
```
Comment (kbdharun) [inline pages.pl/windows/choco-uninstall.md:12]:
```suggestion
`choco uninstall {{pakiet}} --version {{wersja}}`
```

File: pages.pl/windows/choco-pin.md [hunk 21]
```diff
@@ -0,0 +1,21 @@
...
+> Przypięte pakiety są automatycznie pomijane podczas aktualizacji pakietów.
+> Więcej informacji: <https://chocolatey.org/docs/commands-pin>.
+
+- Wyświetlanie listy obecnie przypiętych pakietów wraz z wersjami:
+
+`choco pin list`
+
+- Przypnij pakiet w jego obecnej wersji:
+
+`choco pin add --name {{nazwa_pakietu}}`
```
Comment (kbdharun) [inline pages.pl/windows/choco-pin.md:13]:
```suggestion
`choco pin add --name {{pakiet}}`
```

File: pages.pl/windows/choco-pin.md [hunk 22]
```diff
@@ -0,0 +1,21 @@
...
+
+- Przypnij pakiet w podanej wersji:
+
+`choco pin add --name {{nazwa_pakietu}} --version {{wersja}}`
```
Comment (kbdharun) [inline pages.pl/windows/choco-pin.md:17]:
```suggestion
`choco pin add --name {{pakiet}} --version {{wersja}}`
```

File: pages.pl/windows/choco-pin.md [hunk 23]
```diff
@@ -0,0 +1,21 @@
...
+
+- Odepnij dany pakiet:
+
+`choco pin remove --name {{nazwa_pakietu}}`
```
Comment (kbdharun) [inline pages.pl/windows/choco-pin.md:21]:
```suggestion
`choco pin remove --name {{pakiet}}`
```

File: pages.pl/windows/choco-install.md [hunk 24]
```diff
@@ -0,0 +1,36 @@
...
+
+`choco install {{ścieżka/do/plik_konfiguracyjny.config}}`
+
+- Instalacja podanej specyfikacji nuspec lub pliku nupkg:
+
+`choco install {{ścieżka/do/pliku}}`
+
+- Instalacja konkretnej podanej wersji pakietu:
+
+`choco install {{nazwa_pakietu}} --version {{wersja}}`
```
Comment (kbdharun) [inline pages.pl/windows/choco-install.md:20]:
```suggestion
`choco install {{pakiet}} --version {{wersja}}`
```

File: pages.pl/windows/choco-install.md [hunk 25]
```diff
@@ -0,0 +1,36 @@
...
+
+- Zezwól na instalacjie wielu wersji danego pakietu:
+
+`choco install {{nazwa_pakietu}} --allow-multiple`
```
Comment (kbdharun) [inline pages.pl/windows/choco-install.md:24]:
```suggestion
`choco install {{pakiet}} --allow-multiple`
```

File: pages.pl/windows/choco-install.md [hunk 26]
```diff
@@ -0,0 +1,36 @@
...
+
+- Automatyczna akceptacja wszystkich monitów podczas instalacji (--yes):
+
+`choco install {{nazwa_pakietu}} --yes`
```
Comment (kbdharun) [inline pages.pl/windows/choco-install.md:28]:
```suggestion
`choco install {{pakiet}} --yes`
```

File: pages.pl/windows/choco-install.md [hunk 27]
```diff
@@ -0,0 +1,36 @@
...
+
+- Ustawienie określonego źródła/repozytorium pakietów:
+
+`choco install {{nazwa_pakietu}} --source {{adres_url|alias}}`
```
Comment (kbdharun) [inline pages.pl/windows/choco-install.md:32]:
```suggestion
`choco install {{pakiet}} --source {{adres_url|alias}}`
```

File: pages.pl/windows/choco-install.md [hunk 28]
```diff
@@ -0,0 +1,36 @@
...
+
+- Podanie nazwy użytkownika i hasła do uwierzytelnienia:
+
+`choco install {{nazwa_pakietu}} --user {{nazwa_użytkownika}}} --password {{hasło}}`
```
Comment (kbdharun) [inline pages.pl/windows/choco-install.md:36]:
```suggestion
`choco install {{pakiet}} --user {{nazwa_użytkownika}}} --password {{hasło}}`
```

File: pages.pl/windows/choco-info.md [hunk 29]
```diff
@@ -0,0 +1,20 @@
+# choco info
+
+> Wyświetlanie szczegółowych informacji o pakiecie Chocolatey.
+> Więcej informacji: <https://chocolatey.org/docs/commands-info>.
+
+- Wyświetlanie informacji dotyczących podanego pakietu:
+
+`choco info {{nazwa_pakietu}}`
```
Comment (kbdharun) [inline pages.pl/windows/choco-info.md:8]:
```suggestion
`choco info {{pakiet}}`
```

File: pages.pl/windows/choco-info.md [hunk 30]
```diff
@@ -0,0 +1,20 @@
...
+
+- Wyświetlanie informacji dotyczących podanego pakietu zainstalowanego lokalnie:
+
+`choco info {{nazwa_pakietu}} --local-only`
```
Comment (kbdharun) [inline pages.pl/windows/choco-info.md:12]:
```suggestion
`choco info {{pakiet}} --local-only`
```

File: pages.pl/windows/choco-info.md [hunk 31]
```diff
@@ -0,0 +1,20 @@
...
+
+- Ustawienie określonego źródła/repozytorium pakietów z którego pobrane zostaną informacje:
+
+`choco info {{nazwa_pakietu}} --source {{adres_url|alias}}`
```
Comment (kbdharun) [inline pages.pl/windows/choco-info.md:16]:
```suggestion
`choco info {{pakiet}} --source {{adres_url|alias}}`
```

File: pages.pl/windows/choco-info.md [hunk 32]
```diff
@@ -0,0 +1,20 @@
...
+
+- Podanie nazwy użytkownika i hasła do uwierzytelnienia:
+
+`choco info {{nazwa_pakietu}} --user {{nazwa_użytkownika}} --password {{hasło}}`
```
Comment (kbdharun) [inline pages.pl/windows/choco-info.md:20]:
```suggestion
`choco info {{pakiet}} --user {{nazwa_użytkownika}} --password {{hasło}}`
```

**Review by kbdharun — APPROVED** [review]

**Pull request merged**